Cargo.toml

```diff
@@ -77,8 +77,13 @@ turbopack-wasi = [
 [workspace.lints.clippy]
 too_many_arguments = "allow"
 
-[profile.dev.package.turbo-tasks-macros]
```

Comment:
This config was redundant with the

```
[profile.dev.build-override]
```

below. I measured no meaningful change to build times with/without it.